triggers/run-completion-event-trigger/proto/converters.go

```diff
@@ -0,0 +1,55 @@
+package run_completion_event_trigger
+
+import "github.com/sky-uk/kfp-operator/argo/common"
+
+func ProtoRunCompletionToCommon(protoRunCompletion *RunCompletionEvent) (common.RunCompletionEvent, error) {
```

Comment:
I think the term "domain" makes more sense than "comon"

Reply:
this function name doesn't really fit in with the `ArtifactsConverter` and `StatusConverter` (or the other way round if you prefer).

Reply:
We haven't used `domain` across this repo at all, it wouldn't make sense to just start using it here.

re: ArtifactsConverter and StatusConverter, they were initially private, they are now private again. I think it makes sense for the public function to be more descriptive.